run.go

```diff
@@ -182,7 +182,9 @@ func (b *Builder) Run(command []string, options RunOptions) error {
 	} else if b.Hostname() != "" {
 		g.SetHostname(b.Hostname())
 	}
-	mountPoint, err := b.Mount("")
+	g.SetProcessSelinuxLabel(b.ProcessLabel)
+	g.SetLinuxMountLabel(b.MountLabel)
+	mountPoint, err := b.Mount(b.MountLabel)
```

Comment:
The label we determine in `reserveSELinuxLabels()` isn't recorded except in buildah's own bookkeeping data, so if another tool has already mounted the filesystem, it may have a different context.  Can we detect this?

Reply:
The kernel will refuse to mount the same file system with a different mountlabel.

Reply:
What should we do when that happens?

Reply:
Error out.  Not much you can do but report the error.  We eventually need to get kpod/buildah/cri-o to all agree on the reserved labels to try to prevent it.  

BTW If this is an OverlayMount the label should be unique for the overlay.  you should be able to share the lower, but the upper needs to be unique.